transformation/transformationPlugins/jsonMessageTransformers/jsonJinjavaTransformer/src/test/java/org/opensearch/migrations/transform/JinjavaTransformerTest.java

```diff
@@ -0,0 +1,88 @@
+package org.opensearch.migrations.transform;
+
+import java.util.Map;
+
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.shaded.com.fasterxml.jackson.databind.ObjectMapper;
+
+@Slf4j
+class JinjavaTransformerTest {
```

Comment:
A few comments explaining how this test operates would be helpful.  I find it hard to parse what was actually being tested, where, and how.